MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,138 @@
+quality of the maintainers work is what distinguishes the good projects
+from the great. So please be proud of your work, even the unglamorous
+parts, and encourage a culture of appreciation and respect for *every*
+aspect of improving the project -- not just the hot new features. 
+
+Being a maintainer is a time consuming commitment and should not be
+taken lightly. This document is a manual for maintainers old and new.
+It explains what is expected of maintainers, how they should work, and 
+what tools are available to them.
+
+This is a living document - if you see something out of date or missing,
+speak up!
+
+## What are a maintainer's responsibility?
+
+Part of a healthy project is to have active maintainers to support the
+community in contributions and perform tasks to keep the project running.
+It is every maintainer's responsibility to:
+
+* Keep the community a friendly place
+* Deliver prompt feedback and decisions on pull requests and mailing
+  list threads
+* Encourage other members to help each other, especially in cases the 
+  maintainer is overloaded or feels the lack of needed expertise
+* Make sure the changes made respects the philosophy, design and
+  roadmap of the project
+
+## How are decisions made?
+
+CRIU is an open-source project with an open design philosophy. This
+means that the repository is the source of truth for EVERY aspect of the
+project. *If it's part of the project, it's in the repo. It's in the 
+repo, it's part of the project.*
+
+All decisions affecting CRIU, big and small, follow the same 3 steps:
+
+  * Submit a change. Anyone can do this.
+
+  * Discuss it. Anyone can and is encouraged to do this.
+
+  * Accept or decline it. Only maintainers do this
+
+*I'm a maintainer, should I make pull requests / send patches too?*
+
+Yes. Nobody should ever push to the repository directly. All changes
+should be made through submitting (and accepting) the change.
+
+### Two-steps decision making ###
+
+Since CRIU is extremely complex piece of software we try double hard
+not to make mistakes, that would be hard to fix in the future. In order
+to facilitate this, the "final" decision is made in two stages:
+
+  * We definitely want to try something out
+
+  * We think that the attempt was successful
+
+Respectively, new features get accepted first into the *criu-dev* branch and
+after they have been validated they are merged into the *master* branch. Yet,
+urgent bug fixes may land directly in the master branch. If a change in 
+the criu-dev branch is considered to be bad (whatever it means), then it 
+can be reverted without propagation to the master branch. Reverting from 
+the master branch is expected not to happen at all, but if such an 
+extraordinary case occurs, the impact of this step, especially the question 
+of backward compatibility, should be considered in the most careful manner.
+
+## Who decides what?
+
+All decisions are patches (either in form of pull requests, or sent to 
```

Comment:
Hmm, something does not sound right here:
"All decisions are patches" seems either incomplete or incorrect.

Reply:
*All decisions can be expressed as changes to the repository (either in form of pull requests, or patches sent to the mailing list), ...*